cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/ConsentLibBuilder.java

```diff
@@ -55,6 +57,11 @@ protected StoreClient getStoreClient(){
         return new StoreClient(PreferenceManager.getDefaultSharedPreferences(activity));
     }
 
+    protected ConnectivityManager getManager(){
```

Comment:
`getManager()` is not a good name as it does not indicate what  does it do. Better would be `getConnectivityManager()`